services/github/project/api/routes.py

```diff
@@ -200,7 +210,40 @@ def get_items():
                     item['reviewer_points'] = 0
                 item['num_reactions'] = r.get('reactions').get('totalCount') or 0
                 item['title'] = r.get('title')
-                items.append(item)
+                # check if date is between start and end date
+                if start_creation_date != None and end_creation_date != None:
```

Comment:
Should support just giving one date or the other (e.g., give me all the open issues created before this year).